src/dataDisplay/personPanel/personPanelSummary.jsx

```diff
@@ -577,17 +577,19 @@ function PersonPanelSummary(props) {
                     </Grid.Column>
                 )}
 
-                <Grid.Column
-                    className={personIdColumnClasses}
-                    textAlign="right"
-                >
-                    <Typography
-                        className={classes.personId}
-                        variant="caption"
+                {personId && (
+                    <Grid.Column
+                        className={personIdColumnClasses}
+                        textAlign="right"
                     >
-                        {`Id: ${personId}`}
-                    </Typography>
-                </Grid.Column>
+                        <Typography
+                            className={classes.personId}
+                            variant="caption"
+                        >
+                            {`Id: ${personId}`}
+                        </Typography>
+                    </Grid.Column>
+                )}
```

Comment:
This is not a bad enhancement, although we might double check if omission of Person ID on our comps (e.g. https://www.sketch.com/s/877c6c65-893c-456e-9d33-9bd002036677/a/Vrzbyeo ) is actually intentional.  We might want to show the Person ID, as this is helpful to our end-users I believe.  Let's double check with Sandra and Hannah at scrum today.

Reply:
This would also avoid situations like if personId isn't passed then it would show on UI `personId: undefined`

Reply:
If requirements say it is good, I am fine with this change.

Reply:
Yeah, we brought this up during Team S.H.I.E.L.D. scrum this morning with Garrett and Hannah.  This is a good change for this component, but I do think we might bring back Person ID for our particular usage (and have it wrap to the next line).  You know how our end-users love their Person IDs! :-P